autoscaler/controllers/datacollection/daemonset.go

```diff
@@ -244,26 +245,66 @@ func getDesiredDaemonSet(datacollection *odigosv1.CollectorsGroup,
 							},
 						},
 						{
-							Name: "varlibdockercontainers",
+							Name: "host-dev",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/dev",
+								},
+							},
+						},
+						{
+							Name: "host-etc",
 							VolumeSource: corev1.VolumeSource{
 								HostPath: &corev1.HostPathVolumeSource{
-									Path: "/var/lib/docker/containers",
+									Path: "/etc",
 								},
 							},
 						},
 						{
-							Name: "kubeletpodresources",
+							Name: "host-proc",
 							VolumeSource: corev1.VolumeSource{
 								HostPath: &corev1.HostPathVolumeSource{
-									Path: "/var/lib/kubelet/pod-resources", // TODO: remove this when removing name resoultion processor from collector
+									Path: "/proc",
 								},
 							},
 						},
 						{
-							Name: "hostfs",
+							Name: "host-sys",
 							VolumeSource: corev1.VolumeSource{
 								HostPath: &corev1.HostPathVolumeSource{
-									Path: "/",
+									Path: "/sys",
+								},
+							},
+						},
+						{
+							Name: "host-var",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/var",
+								},
+							},
+						},
+					},
+					InitContainers: []corev1.Container{
+						{
+							Name:  "set-logs-acls",
+							Image: commonconfig.ControllerConfig.CollectorImage,
+							Command: []string{
+								"/bin/sh",
```

Comment:
I'm not sure, but I'll test if we'll need similar selinux changes for openshift (like in odiglet: https://github.com/odigos-io/odigos/blob/1872e358980c16440f65fcaece2a9ea214db5ec9/odiglet/pkg/instrumentation/fs/agents.go#L149)